openedx/core/djangoapps/course_date_signals/admin.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""
+Django Admin pages for SelfPacedRelativeDatesConfig.
+"""
+
+
+from django.contrib import admin
+from django.utils.translation import ugettext_lazy as _
+
+from openedx.core.djangoapps.config_model_utils.admin import StackedConfigModelAdmin
+
+from .models import SelfPacedRelativeDatesConfig
+
+
+class SelfPacedRelativeDatesConfigAdmin(StackedConfigModelAdmin):
```

Comment:
None of this class is unique to the relative dates bit. Why don't we move these bits up a level? These long translated strings are already duplicated in other classes. This class looks a lot like CourseDurationLimitConfigAdmin and ContentTypeGatingConfigAdmin.

Can we consolidate into one super class that lets subclasses just override the interesting bit (list of fields for each fieldset), but shares the boring bits like long strings?

Reply:
Yeah, we potentially. I'll look into it.